src/evaluation/factuality.py

```diff
@@ -133,6 +135,22 @@ def evaluate_summary(
             if not ent["in_source"]:
                 summary_eval["count_entity_extrinsic"] += 1
 
+        # If there are any extrinsic hallucinations, then keep the ratio set to
+        # None to flag that the ratio isn't helpful.
+        #
+        # Alternative could be that if there are no extrinsic hallucinations
+        # that we set the ratio to 1, but we think that unfairly inflates the
+        # the aggregate stat for models that include fewer extrinsic
+        # hallucinations.
+        count_entity_labels = summary_eval["count_entity_label"]
+        count_total_extrinsic_hallucinations = (
+            count_entity_labels['Factual Hallucination']
```

Comment:
Generally I've used the enum `ANNOTATION_LABELS["Factual"], etc.` instead keying by the label directly.

As I'm typing this I realized we can also make `ANNOTATION_LABELS` into a class which has `.Factual`, etc.

Reply:
will update to use `ANNOTATION_LABELS["Factual"]`